server/src/main/java/io/spine/server/event/EventBus.java

```diff
@@ -148,7 +154,28 @@ boolean hasDispatchers(EventClass eventClass) {
 
     @Override
     protected DeadMessageTap<EventEnvelope> getDeadMessageHandler() {
-        return DeadEventTap.INSTANCE;
+        return deadMessageHandler;
+    }
+
+    /**
+     * Creates a {@link DeadMessageTap} instance to handle a dead event by producing an 
+     * {@link UnsupportedEventException}, and saving the event to the Event Store.
+     *
+     * <p>An event gets into this tap if it does not pass the {@link io.spine.server.bus.DeadMessageFilter}.
```

Comment:
Let's simplify this paragraph and follow DRY principle (because now you duplicate your point from the previous paragraph).

```
We must store dead events, as they are still emitted by some entity and therefore are a part of the history for the current bounded context.
```